greek/put_socks_off/put-socks-off.md

```diff
@@ -2,4 +2,5 @@
 άραγε παρόμοια αισθητήρια όπως και εσύ? Βλέπει, ακούει, μυρίζει? 
 Ας δούμε λοιπόν πως αντιδράει στην μοχθηρή μυρωδιά μιας χιλιο-
 φορεμένης καλτσας...
+Τελικά δεν αντιδράει καθόλου... Το μόνο που περιμένει να κάνει είναι να φαγωθεί από κάποιον για να κολυμπήσει στο απέραντο οξύ ενός στομαχιού, χοντρού στομαχιού...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.